scripts/nats-bootstrap/functions.sh

```diff
@@ -108,14 +108,22 @@ nsc edit signing-key -a controllers --sk ${SK_A} \
 	--allow-pubsub '$JS.API.CONSUMER.CREATE.controllers.>' \
 	--allow-pubsub '$JS.API.CONSUMER.MSG.NEXT.controllers.>' \
 	--allow-pubsub '$JS.API.CONSUMER.DELETE.controllers.>' \
-	--allow-pubsub '$JS.API.CONSUMER.CREATE.KV_active-controllers' \
 	--allow-pubsub '$JS.API.CONSUMER.CREATE.KV_firmwareInstall' \
 	--allow-pubsub '$JS.API.CONSUMER.CREATE.KV_firmwareInstall.>' \
 	--allow-pubsub '$JS.API.CONSUMER.DELETE.KV_firmwareInstall.>' \
 	--allow-pubsub '$JS.API.CONSUMER.CREATE.KV_inventory' \
 	--allow-pubsub '$JS.API.CONSUMER.CREATE.KV_inventory.>' \
+	--allow-pubsub '$JS.API.CONSUMER.DELETE.KV_inventory.>' \
+	--allow-pubsub '$JS.API.CONSUMER.CREATE.KV_active-controllers' \
```

Comment:
Vince, can you include some notes on what is the `active-controllers` stream, since we now have two streams I'd like to begin documenting what their role is

Reply:
I have that in code: https://github.com/metal-toolbox/hollow-toolbox/blob/6d1c8212f111898889df40bf5fb23690142a6c9d/events/registry/registry.go#L1